packages/markup/markup.1.0.0/opam

```diff
@@ -2,7 +2,7 @@ opam-version: "2.0"
 
 synopsis: "Error-recovering functional HTML5 and XML parsers and writers"
 
-version: "1.0.0"
+version: "1.0.0-1"
```

Comment:
this will not work as the version in the directory name is going to be taken instead.